src/Consumer/Process.php

```diff
@@ -800,8 +800,8 @@ protected function stateConvert(int $errorCode, ?array $context = null): bool
         }
 
         if ($errorCode === Protocol::OFFSET_OUT_OF_RANGE) {
-            $resetOffset      = $this->getConfig()->getOffsetReset();
-            $offsets          = $resetOffset === 'latest' ? $assign->getLastOffsets() : $assign->getOffsets();
+            $resetOffset = $this->getConfig()->getOffsetReset();
+            $offsets     = $resetOffset === 'latest' ? $assign->getLastOffsets() : $assign->getOffsets();
```

Comment:
I only changed these because they were triggering the Code Quality tests to fail.